core-bundle/src/DataCollector/ContaoDataCollector.php

```diff
@@ -89,6 +103,11 @@ public function getClassesComposerized(): array
         return $data;
     }
 
+    public function getLegacyRouting()
```

Comment:
The return type is missing.

Reply:
Added in a972f8d81da4a457cfd3dcadcb32c77e1d3c88aa